x-pack/plugins/cases/public/components/case_view/components/case_view_activity.test.tsx

```diff
@@ -381,37 +368,14 @@ describe('Case View Page activity tab', () => {
       );
     });
 
-    it('should call user action hooks correctly when filtering for comments', async () => {
-      appMockRender.render(<CaseViewActivity {...caseProps} />);
-
-      expect(await screen.findByTestId('user-actions-activity-bar'));
-      userEvent.click(screen.getByTestId('user-actions-filter-activity-button-comments'));
-
-      expect(screen.getByLabelText(`${userActionsStats.totalComments} active filters`));
-      expect(screen.getByLabelText(`${userActionsStats.total} available filters`));
-      expect(screen.getByLabelText(`${userActionsStats.totalOtherActions} available filters`));
-    });
-
-    it('should show history as active filter correctly', async () => {
-      appMockRender.render(<CaseViewActivity {...caseProps} />);
-
-      expect(await screen.findByTestId('user-actions-activity-bar'));
-      userEvent.click(screen.getByTestId('user-actions-filter-activity-button-history'));
-
-      expect(screen.getByLabelText(`${userActionsStats.totalOtherActions} active filters`));
-      expect(screen.getByLabelText(`${userActionsStats.totalComments} available filters`));
-      expect(screen.getByLabelText(`${userActionsStats.total} available filters`));
-    });
-
     it('should call user action hooks correctly when filtering for history', async () => {
       appMockRender.render(<CaseViewActivity {...caseProps} />);
 
       const lastPageForHistory = Math.ceil(
         userActionsStats.totalOtherActions / userActivityQueryParams.perPage
       );
 
-      expect(await screen.findByTestId('user-actions-activity-bar'));
-      userEvent.click(screen.getByTestId('user-actions-filter-activity-button-history'));
+      userEvent.click(await screen.findByTestId('user-actions-filter-activity-button-history'));
```

Comment:
Some of the tests in the pipeline linked to the issue failed when looking for `'user-actions-filter-activity-button-history'`. I don't know if this test is different enough to prevent it from happening again 😅 at least compared to `'should show history as active filter correctly'`

Reply:
yeah that's why I moved some of them to e2e where we checked only DOM changes, however these 3 tests are checking user actions hook called or not so I wanted to keep it 😄 
I hope removing other tests helps this one to not timeout 🤞

Reply:
yy, no real action is needed, I was just thinking out loud.

> These 3 tests are checking user actions hook called or not so I wanted to keep it 

agree 👍 

Fixing flaky tests is all black magic anyway 🤷